SwiftCryptoTokenFormatter/SwiftCryptoTokenFormatter/TokenFormatter.swift

```diff
@@ -77,7 +77,7 @@ public class TokenFormatter {
     /// ## Short
     /// Short format cuts decimal digits based on the following rules, based on the number's magnitude:
     ///
-    ///   - N < 1000 - displays 5 decimal digits (without rounding)
+    ///   - N < 1000 - displays 5 decimal digits (without rounding); For all values < 0.00001 display '< 0.00001'
```

Comment:
I'd instead make a new line above this line:
```
/// - N < 0.00001 - display '< 0.00001'
```

Reply:
ok